pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -18,10 +20,6 @@ stdenv.mkDerivation rec {
     chmod -x $dest/*.ttf
   '';
 
-  outputHashAlgo = "sha256";
-  outputHashMode = "recursive";
-  outputHash = "0q03gg0sh2mljlbmhamnxz28d13znh9dzca84p554s7pwg6z4wca";
```

Comment:
Yeah, sorry I should have known better :-/ The time stamps were a red herring. The actual problem is that there are two instances of the file `MrsSaintDelafield-Regular.ttf`. One in `ofl/misssaintdelafield` and one in `ofl/mrssaintdelafield`. The one in `ofl/misssaintdelafield` appears to be an older version of the one in `ofl/mrssaintdelafield`. So this patch should hopefully fix the `outputHash` issue:
```patch
diff --git a/pkgs/data/fonts/google-fonts/default.nix b/pkgs/data/fonts/google-fonts/default.nix
index 802c287..e93047a 100644
--- a/pkgs/data/fonts/google-fonts/default.nix
+++ b/pkgs/data/fonts/google-fonts/default.nix
@@ -11,7 +11,19 @@ stdenv.mkDerivation rec {
     sha256 = "0c20vcsd0jki8drrim68z2ca0cxli4wyh1i1gyg4iyac0a0v8wx3";
   };
 
-  phases = [ "unpackPhase" "installPhase" ];
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "0wm09y1mddyg8sr5fp2ckqblw0zi59acwzw7i16alpmspwxishw9";
+
+  phases = [ "unpackPhase" "patchPhase" "installPhase" ];
+
+  patchPhase = ''
+    # Need to remove this directory since it actually contains an
+    # older version of the Mrs Saint Delafield font, which is also
+    # present in `ofl/mrssaintdelafield`. This causes non-determinism
+    # in the install since the installation order is not fixed.
+    rm -rv ofl/misssaintdelafield
+  '';
 
   installPhase = ''
     dest=$out/share/fonts/truetype
```